csrankings-c.csv

```diff
@@ -1221,3 +1221,4 @@ Cyrille Jégourel,SUTD,https://istd.sutd.edu.sg/people/faculty/cyrille-jegourel,
 Cyrille Valentin Artho,KTH Royal Institute of Technology,https://people.kth.se/~artho,wrWmPRQAAAAJ
 Cyrus Omar,University of Michigan,https://people.cs.uchicago.edu/~comar,yiRAVJoAAAAJ
 Cyrus Shahabi,University of Southern California,http://infolab.usc.edu/Shahabi,jEdhxGMAAAAJ
+Chenlin Ma,Shenzhen University,http://csse.szu.edu.cn/en/people?149441,PquGhtEAAAAJ

```

Comment:
FIx web page - http://csse.szu.edu.cn/en/people776b.html?149441

Also, not in alphabetical order.